frontend/src/components/AddDatasetDrawer/AddDatasetDrawer.tsx

```diff
@@ -37,22 +42,45 @@ let AnalyzerButton = withStore('uploadedProvidersFilename')(({ store }) =>
   </div>
 )
 
+function createDataset(store: Store) {
+  let dataSet: Dataset = {
+    dataSources: [
+      store.get('uploadedServiceAreasFilename') || 'No Service Areas',
+      store.get('uploadedProvidersFilename') || 'No Providers'],
+    description: 'Your own data',
+    name: 'Your Data',
+    providers: store.get('providers'),
+    serviceAreaIds: store.get('serviceAreas'),
+    hint: ''
```

Comment:
Could be cool to add an option (in .env?) that populates `hint` with an actual value if enabled.

Reply:
This still needs an extra step before it becomes an actual dataset since datsets live in dataset.ts for now. But yeah, we could even have a popup ask for a description or something in the future

Reply:
Does this dataset get passed to the backend though? If so, caching will work even without adding to datasets.ts as long as the hint is the same when the same files get passed.

Reply:
Nope but that's a good idea as well